src/applications/financial-status-report/pages/income/spouse/benefits/records.js

```diff
@@ -37,9 +36,10 @@ export const uiSchema = {
       education: {
         'ui:title':
           'How much does your spouse get each month for education benefits?',
+        'ui:webComponentField': VaTextInputField,
         'ui:options': {
           classNames: 'schemaform-currency-input',
-          widgetClassNames: 'input-size-3',
+          width: 'md',
```

Comment:
Another spot that I just want to confirm that validation didn't change with the new webComponentField

Reply:
They work!